diff -urNp -x '*.orig' ocamlnet-4.1.5.org/configure ocamlnet-4.1.5/configure
--- ocamlnet-4.1.5.org/configure	2021-03-24 23:42:03.789259823 +0100
+++ ocamlnet-4.1.5/configure	2021-03-24 23:42:03.985919013 +0100
@@ -1263,7 +1263,7 @@ GTK_EXTRA_DEFINES = $gtk2_io_add_watch_s
 # For -enable-apache
 APACHE_MAJOR = $apache_major
 APACHE_LIBDIR = $apache_libdir
-APACHE_OCAMLLIBS = -l$apache_camlrun -ltermcap -lunix -l$libstr
+APACHE_OCAMLLIBS = -l$apache_camlrun -Wl,-R$apache_ocamllibdir -lunix -l$libstr
 APACHE_INCDIR = $apache_incdir
 APACHE_CONFDIR = $apache_confdir
 APACHE_LDFLAGS_SHLIB = $apache_ldflags_shlib
