--- ocamlnet-1.1/src/netstring/Makefile	2005-07-25 22:43:34.000000000 +0000
+++ ocamlnet-1.1.new/src/netstring/Makefile	2005-10-15 15:46:53.600730152 +0000
@@ -101,7 +101,7 @@
 
 .PHONY: mk_netdb
 mk_netdb: tools_done netmappings_iso.pmap netmappings_other.pmap netmappings_jp.pmap
-	mkdir -p netdb
+	mkdir -p $(DESTDIR)netdb
 	cd netdb && \
 		../tools/unimap_to_ocaml/unimap_to_ocaml -netdb ../*.pmap
 
@@ -172,8 +172,8 @@
 
 .PHONY: install-netdb
 install-netdb:
-	mkdir -p $(NET_DB_DIR)
-	cp netdb/*.netdb $(NET_DB_DIR)
+	mkdir -p $(DESTDIR)$(NET_DB_DIR)
+	cp netdb/*.netdb $(DESTDIR)$(NET_DB_DIR)
 	@echo "Installed .netdb files into $(NET_DB_DIR)"
 	if [ "$(INSTMETHOD)" != "findlib" ]; then                     \
 		cat netdb-packlist >>$(LIBDIR)/packlist-$(PKGNAME);   \
