diff -urNp -x '*.orig' ocamlnet-4.1.5.org/configure ocamlnet-4.1.5/configure
--- ocamlnet-4.1.5.org/configure	2017-12-06 21:20:50.000000000 +0100
+++ ocamlnet-4.1.5/configure	2021-03-24 23:42:03.435939925 +0100
@@ -941,7 +941,7 @@ exit 0
 EOF
     # Note: this newer API is never broken in the sense checked below, i.e.
     # such lablgtk2 versions do not exist.
-    if ocamlfind ocamlc -package unix,lablgtk2 -linkpkg -o tmp/gtk tmp/gtktest.ml >/dev/null 2>/dev/null && tmp/gtk; then
+    if ocamlfind ocamlc -package unix,lablgtk2 -linkpkg -o tmp/gtk tmp/gtktest.ml >/dev/null 2>/dev/null; then
         echo "yes"
         gtk2_io_add_watch_supports_lists="-D GTK2_IO_ADD_WATCH_SUPPORTS_LISTS"
     else
