--- ocamlnet-3.7.7/configure~	2014-11-16 11:54:23.000000000 +0100
+++ ocamlnet-3.7.7/configure	2014-11-16 11:57:03.500179144 +0100
@@ -708,7 +708,7 @@
 EOF
     # Note: this newer API is never broken in the sense checked below, i.e.
     # such lablgtk2 versions do not exist.
-    if ocamlfind ocamlc -package unix,lablgtk2 -linkpkg -o tmp/gtk tmp/gtktest.ml >/dev/null 2>/dev/null && tmp/gtk; then
+    if ocamlfind ocamlc -package unix,lablgtk2 -linkpkg -o tmp/gtk tmp/gtktest.ml >/dev/null 2>/dev/null; then
         echo "yes"
         gtk2_io_add_watch_supports_lists="-D GTK2_IO_ADD_WATCH_SUPPORTS_LISTS"
     else
