diff -urNp -x '*.orig' ocamlnet-4.1.5.org/src/rpc-auth-local/rpclocal.c ocamlnet-4.1.5/src/rpc-auth-local/rpclocal.c
--- ocamlnet-4.1.5.org/src/rpc-auth-local/rpclocal.c	2017-12-06 21:20:53.000000000 +0100
+++ ocamlnet-4.1.5/src/rpc-auth-local/rpclocal.c	2021-03-24 23:42:02.899293672 +0100
@@ -8,6 +8,8 @@
 
 #include "config.h"
 
+#define _GNU_SOURCE
+
 #include "caml/mlvalues.h"
 #include "caml/alloc.h"
 #include "caml/memory.h"
