--- ocamlnet-3.3.4/configure~	2011-07-18 11:35:27.050459634 +0200
+++ ocamlnet-3.3.4/configure	2011-07-18 11:37:58.059459600 +0200
@@ -698,7 +698,7 @@
 
 if [ $enable_zip -gt 0 ]; then
     printf "%s" "Checking for camlzip... "
-    if ocamlfind query camlzip >/dev/null 2>/dev/null; then
+    if ocamlfind query zip >/dev/null 2>/dev/null; then
         echo "found"
 	with_netzip=1
     else
--- ocamlnet-3.3.4/src/netzip/META.in~	2011-07-12 17:37:47.000000000 +0200
+++ ocamlnet-3.3.4/src/netzip/META.in	2011-07-18 11:38:49.961459588 +0200
@@ -1,5 +1,5 @@
 version = "@VERSION@"
-requires = "netstring,camlzip"
+requires = "netstring,zip"
 archive(byte) = "netzip.cma"
 archive(native) = "netzip.cmxa"
 archive(native,gprof) = "netzip.p.cmxa"
--- ocamlnet-3.3.4/src/netzip/Makefile~	2011-07-12 17:37:47.000000000 +0200
+++ ocamlnet-3.3.4/src/netzip/Makefile	2011-07-18 11:39:01.235459586 +0200
@@ -8,6 +8,6 @@
 
 include $(TOP_DIR)/Makefile.rules
 
-REQUIRES += camlzip
+REQUIRES += zip
 
 include depend
